classes/system/elasticsearch/server/single.yml

```diff
@@ -19,5 +19,44 @@ parameters:
     server:
       enabled: true
       bind:
-        address: 0.0.0.0
-        port: 9200
\ No newline at end of file
+        address: ${_param:single_address}
+        port: 9200
+      mlockall: true
+      threadpool:
+        bulk:
+          queue_size: 1000
+      gateway:
+        expected_nodes: 1
+        recover_after_nodes: 1
+        recover_after_time: 5m
+      cors:
+        enabled: true
+        allow_origin: '/.*/'
+      cluster:
+        multicast: false
+        minimum_master_nodes: 1
+        members:
+        - host: ${_param:single_address}
+      curator:
```

Comment:
Currently I'm working only on this file. So I'm not sure for the moment what will be common with cluster. But yes I maybe add a common class.